src/services/completions.ts

```diff
@@ -1464,7 +1465,7 @@ namespace ts.Completions {
         }
 
         /**
-         * Gathers symbols that can be imported from other files, deduplicating along the way. Symbols can be “duplicates”
+         * Gathers symbols that can be imported from other files, de-duplicating along the way. Symbols can be "duplicates"

```

Comment:
🤷‍♂ 

![image](https://user-images.githubusercontent.com/3277153/67031415-56aed500-f0c6-11e9-9ab3-2b2b69958e27.png)

Reply:
Haha, I guess code-spell check doesn't have it

Reply:
I make up a lot of words but this wasn’t one of them! 😛